docs/index.html

```diff
@@ -203,10 +203,17 @@ <h5>If you have any legal concerns regarding copyrights or want to report an abu
 
       <div class="search">
         <input type="text" placeholder="Search for icons" ng-model="search">
+        <input type="text" class="result" value="{{ filtered.length }}" title="Number of icons in the current search" disabled>
       </div>
 
+      <ul class="computed-data">
+        <li title="The number of icons. For example: devicon">Total icons: {{ icons.length }}</li>
+        <li title="The number of icon versions. For example: devicon-plain">Total icon version: {{ totalFonts }}</li>
+        <li title="The number of icon versions per icon. Calculated by dividing icon version by icons: {{ avgFonts.toFixed(2) }}">Average icon versions per icon: {{ avgFonts.toFixed(0) }}</li>
```

Comment:
If you want to break the description lines, here goes a suggestion. 😀

```suggestion
        <li title="The number of icons.&#xA;For example: devicon">Total icons: {{ icons.length }}</li>
        <li title="The number of icon versions.&#xA;For example: devicon-plain">Total icon version: {{ totalFonts }}</li>
        <li title="The number of icon versions per icon.&#xA;Calculated by dividing icon version by icons: {{ avgFonts.toFixed(2) }}">Average icon versions per icon: {{ avgFonts.toFixed(0) }}</li>
```

**Example**
*From*
![2023-03-11_21h28_35](https://user-images.githubusercontent.com/76992016/224517621-37453a3f-c19f-4977-99c6-3e5b16d1de29.png)
*To*
![2023-03-11_21h28_01](https://user-images.githubusercontent.com/76992016/224517623-6117e20b-bf03-4247-af38-0714c28a9157.png)